humblebundle_downloader/download_library.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, library_path, cookie_path=None, cookie_auth=None,
         self.purchase_keys = purchase_keys
         self.trove = trove
         self.update = update
+        self.content_types = ['web'] if content_types is None else list(map(str.lower, content_types))  # noqa: E501
```

Comment:
Would it make sense to validate the types against the known list? That would protect users from typos. But also prevent downloading novel types if they ever become available (or already are?).